kubectl-plugin/test/e2e/kubectl_ray_log_test.go

```diff
@@ -0,0 +1,141 @@
+package e2e
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestKubectlRayLog(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Kubectl Ray Get")
+}
+
+var _ = Describe("Calling ray plugin `log` command on Ray Cluster", Ordered, func() {
+	It("succeed in retrieving all ray cluster logs", func() {
+		expectedDirPath := "./raycluster-sample"
+		expectedOutputStringFormat := `No output directory specified, creating dir under current directory using resource name\.\nCommand set to retrieve both head and worker node logs\.\nDownloading log for Ray Node raycluster-sample-head-\w+\nDownloading log for Ray Node raycluster-sample-small-group-worker-\w+`
+
+		cmd := exec.Command("kubectl", "ray", "log", "raycluster-sample", "--node-type", "all")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).NotTo(HaveOccurred())
+		Expect(strings.TrimSpace(string(output))).Should(MatchRegexp(expectedOutputStringFormat))
+
+		// Check that the log directory exists
+		logDirInfo, err := os.Stat(expectedDirPath)
```

Comment:
@andrewsykim are there files that we want to make sure gets retrieved? Eg stdout.log

Reply:
It would be good to check the contents of some log files that exist in every standard Ray cluster

Reply:
It checks for "stdout.log" and "raylet.out" and a specific log content. PTAL!

Reply:
A really good test would be submitting a job in the cluster and then checking that the job logs are present when running `kubectl ray logs`